newIDE/app/src/BehaviorsEditor/NewBehaviorDialog.js

```diff
@@ -340,7 +340,7 @@ export default function NewBehaviorDialog({
                 </Line>
                 <Line justifyContent="center" alignItems="center">
                   <FlatButton
-                    leftIcon={<Create />}
+                    leftIcon={<Edit />}
```

Comment:
just to confirm Create/Edit is the right icon?

Reply:
Yes, that's (almost) the same
<img width="63" alt="image" src="https://user-images.githubusercontent.com/32449369/227579955-f42ba191-a3a7-46e1-a68a-44ab53d9dca5.png">
<img width="79" alt="image" src="https://user-images.githubusercontent.com/32449369/227579959-1dca5177-e7a2-4c42-a75e-7d72e1968b16.png">